mypy/semanal.py

```diff
@@ -3734,13 +3751,23 @@ def name_not_defined(self, name: str, ctx: Context) -> None:
         if extra:
             message += ' {}'.format(extra)
         self.fail(message, ctx)
+
         if 'builtins.{}'.format(name) in SUGGESTED_TEST_FIXTURES:
             # The user probably has a missing definition in a test fixture. Let's verify.
             fullname = 'builtins.{}'.format(name)
             if self.lookup_fully_qualified_or_none(fullname) is None:
                 # Yes. Generate a helpful note.
                 self.add_fixture_note(fullname, ctx)
 
+        if name.lower() in TYPES_FOR_UNIMPORTED_HINTS:
+            # User probably forgot to import these types.
+            unimported_module, suggestion = TYPES_FOR_UNIMPORTED_HINTS[name.lower()]
+            hint = (
+                f'Did you forget to import it from "{unimported_module}"?'
```

Comment:
Unfortunately we can't use f-strings since mypy still supports Python 3.5.

Reply:
:+1: